infer_structcol/inference.py

```diff
@@ -22,25 +23,41 @@ def find_max_like(data, sample, seed=None):
         experimental dataset
     sample: Sample object
         information about the sample that produced data_spectrum
-    phi_guess: float
-        user's best guess of the expected volume fraction
     seed: int (optional)
-        if specified, passes the seed through to the MC multiple scatterin 
+        if specified, passes the seed through to the MC multiple scattering 
         calculation
-
+    theta_guess: list of floats (optional)
+        user's best guess of the expected parameter values (phi, radius, 
+        thickness)
+    theta_range: 2 by 3 array of floats 
+        user's best guess of the expected ranges of the parameter values 
+        ([[min_phi, max_phi], [min_radius, max_radius], [min_thickness, max_thickness]]) 
+    
     Returns
     -------
-    theta: 3 or 5 -tuple
-        best fit (phi, l0, l1) or (phi, l0, l1, l0, l1) as floats
+    theta: 5 or 7 -tuple
+        best fit (phi, radius, thickness, l0, l1) or (phi, radius, thickness, 
+        l0, l1, l0, l1) as floats
     '''
+    if theta_guess == None:
+        phi_guess = 0.55
+        radius_guess = 120.
+        thickness_guess = 120.
+    else:
+        phi_guess = theta_guess[0]
+        radius_guess = theta_guess[1]
+        thickness_guess = theta_guess[2]
+    
     def resid(params):
```

Comment:
This is a separate concern because here we are converting a dictionary into a tuple. We should be able to avoid running ``if reflectance and transmittance; elif reflectance; else``. This is a bit messy because it is not clear from the code that ``else`` refers to transmittance. It also involves repeating code for adding phi, radius, and thickness. I think the code I wrote in my comment should still work.